### DISCUSSION TITLE
List contributor names in citation

### DISCUSSION THREAD

**Discussion by KellyStathis** [datacite/datacite-suggestions#43] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

This would improve the contributors' visibility

### What solution might meet your needs?

Names in contributors field should be listed in data citation

### What alternatives have you tried or considered?

A value that doesn't quite fit, adding contributors as co-creators

### Is there anything else you would like to share?

*Suggestion was previously submitted by a user. For DataCite staff reference: https://docs.google.com/spreadsheets/d/1ItM7bYocla83TCYmHI9oZ9Kgt7X0Lznfv6Zls7Gmi88/edit?resourcekey=&gid=1473745305#gid=1473745305*

### What group(s) would benefit from your suggestion?

- [ ] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by tedhabermann** [comment]
This verges on adding "author" as a contributorType and dropping creators - a great idea!
